enhancements/proposals/external_seeds.md

```diff
@@ -0,0 +1,145 @@
+
+## Proposal
+
+This document proposes extending the existing ScyllaCluster CR with a new field in its Specification which allows the users to control the external seeds provided to ScyllaDB processes.
+The changes required for ScyllaDB Operator and the sidecar to support the enhancement are also described.
+In addition to that, the document also proposes the procedure for compiling the list of seeds propagated to ScyllaDB processes when external seeds are provided.
+
+### User Stories
+
+#### As a ScyllaDB cluster administrator:
+- I want to control the list of external seeds provided to ScyllaDB nodes.
+- I want to create a multi datacenter ScyllaDB by manually providing the external seeds to chosen datacenters using the ScyllaCluster CR.
+- I want to add a datacenter to existing ScyllaDB datacenters by manually providing the external seeds to chosen datacenters using the ScyllaCluster CR.
+
+### Notes/Constraints/Caveats [Optional]
+
+#### Current role of seeds in ScyllaDB
+
+With https://github.com/scylladb/scylladb/pull/6848, the role of seeds in ScyllaDB has been reduced to initial point of contact nodes.
+Hence they do not have any additional significance past bootstrap, and ScyllaDB nodes with external seeds provided are not prone to any additional issues related to e.g. lack of connectivity between DCs past that point.
+
+#### Multi DC setup procedure
+
+[Create a ScyllaDB Cluster - Multi Data Centers (DC)](https://opensource.docs.scylladb.com/branch-5.3/operating-scylla/procedures/cluster-management/create-cluster-multidc.html#procedure) procedure in ScyllaDB documentation explicitly states to use IP address of the first node, *and only the first node*, as seed for all nodes in all datacenters.
+This has proven to be incorrect in two ways: 
+1. Domain name addresses are also accepted as seeds. 
+2. There is no special significance to the first node of the first DC - it is just a simplification used for the purpose of minimising the risk of creating multiple clusters instead of a single, multi DC cluster.
+
+The specified constrains hold - the datacenters need a matching cluster name and distinct datacenter and rack names.
+
+### Risks and Mitigations
+
+The proposed enhancement does not carry any risks for the functionality or stability of the operator.
+
+See [Drawbacks](#drawbacks) section for potential risks for end users.
+
+#### 
+
+## Design Details
+
+### ScyllaCluster API
+
+The following extension to the existing ScyllaCluster CR API is proposed:
+```go
+// ScyllaClusterSpec defines the desired state of Cluster.
+type ScyllaClusterSpec struct {
+    ...
+	
+	// externalSeeds specifies the external seeds to propagate to ScyllaDB binary on startup.
+	// +optional
+	ExternalSeeds []string `json:"externalSeeds,omitempty"`
+}
+```
+
+The provided `externalSeeds` are passed to the sidecar through a new `--external-seeds` flag. StatefulSet reconciliation in ScyllaDB operator is modified accordingly.
```

Comment:
https://github.com/scylladb/scylladb/blob/6c66030b7bbcf3ef5d962a0f4ca48d46582ca3a2/gms/gossiper.cc#L947 Scylla falls back to using initial contact points in case it can't reach any other nodes, so I believe providing the option to update externalSeeds seems like a failsafe which we might not want to get rid of. Wdyt? Also I'll add this information to the proposal.